pkg/server/plugin/nodeattestor/x509pop/x509pop_test.go

```diff
@@ -158,7 +160,9 @@ func (s *Suite) TestAttestFailure() {
 
 	s.T().Run("not configured", func(t *testing.T) {
 		attestor := new(nodeattestor.V1)
-		plugintest.Load(t, BuiltIn(), attestor)
+		plugintest.Load(t, BuiltIn(), attestor,
```

Comment:
can you add test case where no trustedbundle is returned? 
and how about defaults?
are

Reply:
Struggling a bit with how to write the tests....

Seems like I need to create a new cert in ./test/fixture/nodeattestor/x509pop/generate.go with the url based spiffe id to test with? There are some magic numbers in that file. Are they just arbitrary, or are there some reason for the specific numbers chosen?

Reply:
Reading through it some more, it looks like they just have patterns in it. so rather arbitrary.

Reply:
Ok. I have tests for no trust bundle, along with a valid svid, an invalid svid, and a no svid.